keps/3589-manage-jobs-selectively/README.md

```diff
@@ -110,6 +117,16 @@ managedJobsNamespaceSelector:
   matchLabels:
     kueue.x-k8s.io/managed-namespace: true
 ```
+#### Story 2
+
+Cluster admins want to opt in only selected namespaces to Kueue management. A user in a namespace not selected by managedJobsNamespaceSelector should not be able to submit a Kueue-managed workload by manually setting the queue-name label. This change ensures that only workloads in opted-in namespaces are ever reconciled, regardless of how they are labeled.
+
+Cluster admin has to label namespaces that are subject to management in order for the reconciliation to work.
+```yaml
+managedJobsNamespaceSelector:
+  matchLabels:
+    kueue.x-k8s.io/managed-namespace: true
```

Comment:
```suggestion
    example.com/sample: true
```

Do you indicate introducing this label? If not, could you avoid using the `kusus.x-k8s.io` domain?

Reply:
Does that mean the ManagedJobsNamespaceSelector introduces the `kueue.x-k8s.io/managed-namespace` label?
Why do we need to introduce the dedicated static label, `kueue.x-k8s.io/managed-namespace`?